frontend/views/containers/chatroom/TouchLinkHelper.vue

```diff
@@ -0,0 +1,230 @@
+<template lang='pug'>
+.c-touch-link-helper(:class='{ "is-active": ephemeral.isActive }')
+  .c-overlay-background(@click='close')
```

Comment:
We do have [`vue-clickaway`](https://github.com/simplesmiler/vue-clickaway), but the overlay makes more sense because it dims the background 👍

Reply:
Yes and Plus, it also allows us to dismiss the component when any point on the background is pressed.